website/docs/execution-node/fee-recipient.md

```diff
@@ -59,21 +59,21 @@ A fee recipient address can be configured on your client instance by using one o
     <td>
     Sets a default ETH address for all validator public keys. <br /> <br /> 
     <strong>Example</strong>: <code>--suggested-fee-recipient=0x0123456722E6b0000012BFEBf6177F1D2e9758D9</code> <br /> <br /> 
-    <strong>Note</strong>: This setting overrides the two config options below. If you set this, the config options will be ignored.
+    <strong>Note</strong>: This setting is overwritten by the flags below and will act as a fallback catch all.
     </td>
   </tr>
   <tr>
-    <td><code>fee-recipient-config-file</code></td>
+    <td><code>proposer-settings-file</code></td>
     <td>
-    Sets the local file location for the fee recipient JSON configuration. <br /> <br /> 
-    <strong>Example</strong>: <code>--fee-recipient-config-file=./fee_recipient_config.json</code> <br /> <br /> 
+    Sets the local file location for the proposer-settings ( settings for your validator key when proposing a block) in YAML or JSON file formats. <br /> <br /> 
+    <strong>Example</strong>: <code>--proposer-settings-file=./fee_recipient_config.json</code> <br /> <br /> 
     </td>
   </tr>
   <tr>
-    <td><code>fee-recipient-config-url</code></td>
+    <td><code>proposer-settings-url</code></td>
     <td>
-    Sets a URL for a remote fee recipient JSON configuration.  <br /> <br /> 
-    <strong>Example</strong>: <code>--fee-recipient-config-url=http://example.com/api/getFeeRecipient</code> <br /> <br /> 
+    Sets a URL for remote proposer-settings( settings for your validator key when proposing a block).  <br /> <br /> 
```

Comment:
```suggestion
    A remote `proposer-settings` configuration endpoint in URL format. <br /> <br /> 
```

Removing a bit of duplication since we provide context above - once you're ready for an all-up review we can take a look at the rendered content to see how it flows with/without the duplication?